parm/config/gfs/config.ice

```diff
@@ -2,4 +2,8 @@
 
 echo "BEGIN: config.ice"
 
+# Override atm-only FV3 settings when ice model is on
+export min_seaice="1.0e-6"
```

Comment:
Both of these are variables set in input.nml .  Do the tests that were run confirm that there are no answer changes?

Reply:
I can confirm the namelists use these settings. I have not checked the output for bitwise indenticality.

Reply:
Okay, it'd might be good to check for bitwise identity too and to make sure this isn't being sourced for atm-only case as mentioned in the comments.  Right now I can't seem to find where the sourcing of this config is happening - but perhaps the logic has just changed and I haven't had time to understand but I don't see where this config is sourced for the forecast job if coupled.    I'll try to test this tomorrow unless you have saved output that we can compare results with.

Reply:
It gets sourced by config.fcst based on the DO_<COMPONENT> switches:

https://github.com/NOAA-EMC/global-workflow/blob/527aa16d8e003ddb65e6ed4e1b5b47fb79fa5921/parm/config/gfs/config.fcst#L23-L29

Reply:
Was finally able to check the files and they are bitwise identical.

Reply:
Thanks for doing this check @WalterKolczynski-NOAA !!!!